alexandria/core/filters.py

```diff
@@ -67,3 +67,12 @@ class FileFilterSet(FilterSet):
     class Meta:
         model = models.File
         fields = ["original", "renderings", "type", "meta"]
+
+
+class TagFilterSet(FilterSet):
+    meta = JSONValueFilter(field_name="meta")
+    category = CharFilter(field_name="documents__category__slug")
```

Comment:
I'm not sure about the intention behind this filter: You're searching for tags with documents in the given category? What's the use case here?

Reply:
Use case: A user has selected a category and wants to filter the documents in this category based on tags. For this, we only want to present the tags that have at least one document tagged with it in this category.

I have to admint, that the name `category` for this filter is not very descriptive. An alternative could be `documents-category-slug`, but that's rather vebose. Do you have something in mind?

Reply:
Aah okay that makes sense. Maybe something along the line of `with-documents-in-category`? I don't really mind the verbosity tbh, we're not on DOS anymore with 8 char names :stuck_out_tongue:

Reply:
I went with `with-documents-in-category`.